### PR TITLE
Cleanup System.Reflection.TypeExtensions.MethodInfo tests

### PR DESCRIPTION
**Pull request by hughbe**
/cc @stephentoub

### CODE REVIEW

File: src/System.Reflection.TypeExtensions/tests/MemberBase/MethodBaseContainsGenericParameters.cs [hunk 1]
```diff
@@ -1,227 +0,0 @@
...
-            }
-        }
-
-        // Positive Test 1: ContainsGenericParameters should return true for open generic method
-        [Fact]
-        public void PosTest1()
-        {
-            Type type = typeof(TestClass);
-            MethodBase methodInfo = type.GetMethod("TestGenericMethod");
-            Assert.True(methodInfo.ContainsGenericParameters, "ContainsGenericParameters returns false for open generic method" + methodInfo);
```
Comment (stephentoub) [inline src/System.Reflection.TypeExtensions/tests/MemberBase/MethodBaseContainsGenericParameters.cs]: Where did these ContainsGenericParameters tests go?

File: src/System.Reflection.TypeExtensions/tests/MethodInfo/GetBaseDefinition.cs [hunk 2]
```diff
@@ -1,330 +0,0 @@
...
-
-            type = Type.GetType("System.Reflection.Tests.Co4611_a3");
-            Assert.NotNull(type);
-            Assert.Equal("Co4611_a3", type.Name);
-
-            mInfo = type.GetMethod("MethodA", new Type[1] { typeof(System.String) });
-            Assert.NotNull(mInfo);
-
-            obj = new Co4611_a3();
-            var = mInfo.Invoke(obj, (Object[])(new string[] { "test string" }));
```
Comment (stephentoub) [inline src/System.Reflection.TypeExtensions/tests/MethodInfo/GetBaseDefinition.cs]: Where did these invocation tests go?

**Comment by stephentoub** [comment]
A few things appear to have been removed and I'm not sure why.  Otherwise LGTM.

File: src/System.Reflection.TypeExtensions/tests/MemberBase/MethodBaseContainsGenericParameters.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Reflection.TypeExtensions/tests/MemberBase/MethodBaseContainsGenericParameters.cs]: Oh I forgot to write this down in the PR description.

I rebased PR #11484, and git didn't delete the old member base file.

You can see the old tests here: https://github.com/dotnet/corefx/blob/master/src/System.Reflection.TypeExtensions/tests/MethodBaseTests.cs#L12

File: src/System.Reflection.TypeExtensions/tests/MethodInfo/GetBaseDefinition.cs (hunk 2 above)
Comment (hughbe) [inline src/System.Reflection.TypeExtensions/tests/MethodInfo/GetBaseDefinition.cs]: This is the only invocation test, but I will add it back

Comment (hughbe) [inline src/System.Reflection.TypeExtensions/tests/MethodInfo/GetBaseDefinition.cs]: added back

**Comment by stephentoub** [comment]
Thanks, @hughbe.

**Pull request merged**